### ISSUE TITLE
Request for inclusion: BCA Feature Selection

### ISSUE THREAD

**Issue by aminzarshenas** [scikit-learn-contrib/scikit-learn-contrib#32]
Request for project inclusion in scikit-learn-contrib

* Project name: bca
* Project description: Binary Coordinate Ascent (BCA) algorithm for feature subset selection
* Authors: Amin Zarshenas, Vijay Srinivas Tida, Kenji Suzuki
* Current repository: https://github.com/aminzarshenas/bca
* Requirements:
 * [ ] scikit-learn compatible (check_estimator passed)
    It does not pass the check_estimator (similar to RFE from sklearn.feature_selection.rfe).
    It passes our test cases.
 * [x] Documentation (guide, API reference, example gallery)
 * [x] Unit tests (coverage: xx%)
 * [x] Python3 compatible
 * [x] PEP8 compliant
 * [x] Continuous integration

Brief description:

Binary Coordinate Ascent (BCA) is an optimization algorithm for wrapper feature subset selection (FSS) for classification. As an advantage to other wrapper FSS meta-heuristics, e.g., sequential forward selection (SFS) and sequential floating forward selection (SFFS), BCA is computationally significantly more efficient while maintaining the classification performance for unseen data. BCA can be used for binary or multi-class classification problems and it can be coupled with different classifiers such as SVM, Naive Bayes, etc.

**Comment by agramfort** [comment]
where is the documentation? I don't see the url.

>

**Comment by aminzarshenas** [comment]
The documentation is not formal at the moment. The usage is in the readme.md file. Is there a guideline or common style to create the documentation?

**Comment by MaxHalford** [comment]
You're supposed to follow the official [template](https://github.com/scikit-learn-contrib/project-template). It uses [sphinx](http://www.sphinx-doc.org/en/master/) to generate documentation from your code comments.

**Comment by aminzarshenas** [comment]
Thank you for your help and the links. I followed the template and put the .rst files as well as the built .html files in the bca/doc directory.

**Comment by agramfort** [comment]
or use gh-pages

**Comment by aminzarshenas** [comment]
@MaxHalford @agramfort Thank you for your help and advice and sorry for the delay as I was not very familiar with sphinx and readthedocs. I modified a bit, created the [documentation](https://bca.readthedocs.io) and put the link in the README.md file as well.
